fortuna/src/config/register_provider.rs

```diff
@@ -35,7 +34,7 @@ pub struct RegisterProviderOptions {
     /// The fee to charge (in wei) for each requested random number
     #[arg(long = "pyth-contract-fee")]
     #[arg(default_value = "100")]
-    pub fee: U256,
+    pub fee: u128,
```

Comment:
this is fallout from updating the ABI. this changed on-chain in a previous pr and didn't get updated here